.github/CODEOWNERS

```
@@ -157,3 +157,10 @@
 /src/permission/* @nodejs/security-wg
 /doc/api/permissions.md @nodejs/security-wg
 /lib/internal/process/permission.js @nodejs/security-wg
+
+# Dependency Update Tools
+
+/tools/dep_updaters/* @nodejs/security-wg
+/.github/workflows/tools.yml @nodejs/security-wg
+/.github/workflows/update-v8.yml @nodejs/security-wg @nodejs/v8-update
+/.github/workflows/update-openssl.yml @nodejs/security-wg
```

Nit: Alphabetical order.

```suggestion
/.github/workflows/tools.yml @nodejs/security-wg
/.github/workflows/update-openssl.yml @nodejs/security-wg
/.github/workflows/update-v8.yml @nodejs/security-wg @nodejs/v8-update
/tools/dep_updaters/* @nodejs/security-wg
```

---

https://github.com/nodejs/node/pull/48124 does it for the rest of the file.